packages/react-native-web/src/exports/Button/index.js

```diff
@@ -10,9 +10,9 @@
  * @flow
  */
 
-import ColorPropType from '../../propTypes/ColorPropType';
-import StyleSheet from '../../apis/StyleSheet';
-import TouchableOpacity from '../Touchable/TouchableOpacity';
+import ColorPropType from '../ColorPropType';
+import StyleSheet from '../StyleSheet';
+import TouchableOpacity from '../TouchableOpacity';
```

Comment:
More obvious that `Button` is made of simpler primitives and not part of the "core"